editor/editor_node.cpp

```diff
@@ -179,13 +179,112 @@
 
 EditorNode *EditorNode::singleton = nullptr;
 
+void EditorNode::disambiguate_filenames(Vector<String> &filenames, const Vector<String> full_paths) {
+	// Keep track of a list of "index sets," i.e. sets of indices
+	// within disambiguated_scene_names which contain the same name
+	// We must free the Set<int>*'s
+	Vector<Set<int>> index_sets;
+	Map<String, int> scene_name_to_set_index;
+	for (int i = 0; i < filenames.size(); i++) {
+		String scene_name = filenames[i];
+		if (!scene_name_to_set_index.has(scene_name)) {
+			index_sets.append(Set<int>());
+			scene_name_to_set_index.insert(filenames[i], index_sets.size() - 1);
+		}
+		index_sets.write[scene_name_to_set_index[scene_name]].insert(i);
+	}
+
+	// For each index set with a size > 1, we need to disambiguate
+	for (int i = 0; i < index_sets.size(); i++) {
+
```

Comment:
We updated our style guide so now empty space at the start of code blocks should be removed.